edx_django_utils/monitoring/internal/backends.py

```diff
@@ -62,6 +62,14 @@ def create_span(self, name):
         or create a new root span if not currently in a span.
         """
 
+    @abstractmethod
+    def tag_root_span(self, exception):
```

Comment:
Maybe rename to `tag_root_span_with_error`, or something like that? This reads like you will be adding general span tags to the root span.